1-starter/README.md

```diff
@@ -36,18 +36,60 @@
   </p>
 </div>
 
-# 📜 Documentation
+## 🏗 Development
 
-> **[discord-ts.js.org](https://discord-ts.js.org)**
+```
+npm install
+npm run dev
+```
 
-> **[Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)**
+If you want to use [Nodemon](https://nodemon.io/) to auto-reload while in development:
 
-# ☎️ Need help?
+```
+npm run watch
+```
+
+## 💻 Production
+
+```
+npm install --production
+npm run build
+npm run start
+```
+
+### 🐋 Docker (Recommended)
+
+To start your application:
+
+```
+docker-compose up -d
+```
+
+To shut down your application:
+
+```
+docker-compose down
+```
+
+To view your application's logs:
+
+```
+docker-compose logs
+```
+
+For the full command list please view the [Docker Documentation](https://docs.docker.com/engine/reference/commandline/cli/).
+
+## 📜 Documentation
+
+- [discord-ts.js.org](https://discord-ts.js.org)
+- [Tutorials (dev.to)](https://dev.to/oceanroleplay/series/14317)
+
+## ☎️ Need help?
 
 - [Check frequently asked questions](https://discord-ts.js.org/docs/faq)
 - [Check examples](https://github.com/oceanroleplay/discord.ts/tree/main/packages/discordx/examples)
-- Ask in **[discord server](https://discord-ts.js.org/discord)**
```

Comment:
revert this as well

Reply:
Sounds too stiff...

Reply:
> Sounds too stiff...

Keep it in, but just reword it to sound more inviting. I agree with Harry we need to have our discord server in

Reply:
This never got removed, It got changed it too `Join and ask in our Discord Server` which sounds like the best way to put it without changing it to much.

Reply:
Revert